documentation/5_Organizational_Agreements.md

```diff
@@ -65,4 +65,162 @@ process is defined by the IDSA certification working group.
+semantic interoperability aspects, while the Data Space Governance
+Authority defines the organizational and legal aspects.
+Based on the governance scheme, the participants participate in the
+defintion of the policies and rules of a data space, including the
+framework or agreements on all four levels.
+
+### Technical level
+
+The technical level covers the software and hardware components for
+controlled, sovereign and secure sharing of data. It consists of five
+sublevels with the topics that require adequate governance:
+
+### Governance instruments
+
+The various governance instruments that may be considered by the IDSA
+are listed in the Table below. These governance tools each describe a
+different aspect of a certain activity and are used in the next section
+to provide a multidimensional overview of the intra and inter data
+spaces governance.
+
+| **Governance Instrument** | **Governance instrument description**      |
+|--------------------------------- |---------------------------------|
+| *Standardization* | Ensuring that the tasks, processes, and guidelines around this activity are formalized, documented, and  aligned between data spaces instances and the IDSA. The standardization efforts can generally be used as a  blueprint or starting point for the stakeholders.|
+| *Certification* | Validating that stakeholders act according to the standardized way of working. Certification is divided into component certification (certification of technical/software components) and organizational certification (organizational and legal processes).    |
+| *Development* | The activity may require (software) development tasks as part of the realization, which should be compliant with the standardization activities and best followed by a certification. |
+| *Operations*           | The operations are about the exploitation and usage of the developed components. The operations can be certified as part of the organizational certification. |
+| *Communication*        | The dissemination of the activity is an important aspect which might include the awareness of the standardization and certification, but also contains marketing aspects. |
+| *Support*   | The support activities include the structured assistance from stakeholders involved in the operation, development, certification, and communication activities. |
+
+### Governance for inter data space interoperability
+
+> update text
+
+There will be no single data space. Individual sectors or communities
+are expected to develop their own data space instances. Being able to
+seamlessly share data over these data space instances brings clear
+benefits. It extends the reach and scope of accessible data and allows
+the development of new business models and services across sectors and
+regions. Interoperability between data space instances adds major value,
+resulting in a federation of interoperable data spaces.
+
+The federation of data spaces require a common set of reusable standards
```

Comment:
Federations (plural) as it is extremely unlikely that there will be one federation of dataspaces to govern them all.